hadoop-ozone/dist/src/shell/conf/db.ini.template

```diff
@@ -0,0 +1,25 @@
+# This is a RocksDB option file.
```

Comment:
We should include a license header in this file to pass the RAT check.
Is this file copied from RocksDB?

Reply:
> We should include a license header in this file to pass the RAT check.
> Is this file copied from RocksDB?

Copied from https://github.com/facebook/rocksdb/blob/main/examples/rocksdb_option_file_example.ini, and added some description

Reply:
Maybe we should keep the original author, and add this file in LICENSE.
Cc @adoroszlai, what do you think?

For example 
https://github.com/apache/ozone/blob/57d66137fa54d566752f2787e853deecda9d194b/LICENSE.txt#L216-L217